PocketKit/Sources/Sync/PersistentContainer.swift

```diff
@@ -9,6 +9,7 @@ public class PersistentContainer: NSPersistentContainer {
 
     private lazy var backgroundContext = {
         let context = newBackgroundContext()
+        context.mergePolicy = NSMergePolicy.mergeByPropertyObjectTrump
```

Comment:
substance

Reply:
We are operating on tags in multiple NSManagedObjectContexts, we need to specify how they get merged.

Reply:
this applies to all objects as opposed to just tags right? also thought merging was handle via 
`context.automaticallyMergesChangesFromParent = true`

I'm probably not understanding what's going on, so we can discuss this live at a later date. thanks!